index.py

```diff
@@ -13,13 +13,13 @@ def get(self):
         return "Nav-E API", 200
     
 def abort_trip_dne(tripID):
-    # not sure why I get an error here "HTTPException.__init__() got an unexpected keyword argument 'message'"
+    # not printing my message -> keyError: 0
```

Comment:
Your trips dict doesn't have key 0. And you seem to be requesting trip 0 in your http call